README.md

```diff
@@ -2,104 +2,91 @@
+The Docker extension makes it easy to build, manage and deploy containerized applications from Visual Studio Code.
 
-**Visit the [wiki](https://github.com/Microsoft/vscode-docker/wiki) for additional information about the extension.**
+**Check out [Working with containers](https://code.visualstudio.com/docs/containers/overview) topic on Visual Studio Code documentation site to get started**.
 
-## Prerequisites
+[The Docker extension wiki](https://github.com/Microsoft/vscode-docker/wiki) has troubleshooting tips and additional technical information.
 
-Install [Docker](https://aka.ms/AA37qtj) on your machine and add it to the system path.
+## Installation
 
-On Linux, you must also follow the steps in “Manage Docker as a non-root user” from [Post-installation steps for Linux](https://aka.ms/AA37yk6) because VS Code runs as a non-root user.
+[Install Docker](https://docs.docker.com/install/) on your machine and add it to the system path.
 
-## Features
+On Linux, you should also [enable Docker CLI for the non-root user account](https://docs.docker.com/install/linux/linux-postinstall/#manage-docker-as-a-non-root-user) that will be used to run VS Code.
 
-* Automatic `Dockerfile`, `docker-compose.yml`, and `.dockerignore` file generation (Open the command Palette, <kbd>F1</kbd> by default, and search for `Docker: Add Docker files to Workspace`)
-* Syntax highlighting, hover tips, IntelliSense (completions) for `docker-compose.yml` and `Dockerfile` files
-* Linting (errors and warnings) for `Dockerfile` files
-* Command Palette (<kbd>F1</kbd> by default) integration for the most common Docker commands (for example `docker build`, `docker push`, etc.)
-* Explorer integration for managing images, containers, registries, and more
-* Deploy images from a registry directly to Azure App Service
-* [Debug .NET Core applications](https://aka.ms/AA6jdzn) running in Linux Docker containers
+To install the extension, open the Extensions view, search for `docker` to filter results and select Docker extension authored by Microsoft.
 
-### Generating Docker Files
+## Overview of the extension features
 
-Press <kbd>F1</kbd> (by default) and search for `Docker: Add Docker Files to Workspace` to generate `Dockerfile`, `docker-compose.yml`, `docker-compose.debug.yml`, and `.dockerignore` files for your workspace type:
+### Editing Docker files
 
-![dockerfile](resources/readme/generateFiles.gif)
+You can get IntelliSense when editing your `Dockerfile` and `docker-compose.yml` files, with completions and syntax help for common commands.
 
-> Note: The `docker-compose.yml` and `docker-compose.debug.yml` files are not generated for all platforms, such as .NET Core applications, and are optional for Node.js applications.
+![IntelliSense for Dockerfiles](resources/readme/dockerfile-intellisense.png)
 
-### Editing
+In addition, you can use the Problems panel (`kb(workbench.actions.view.problems)`) to view common errors for `Dockerfile` and `docker-compose.yml` files.
```

Comment:
This shows like below. Is this right?
![image](https://user-images.githubusercontent.com/1775825/74072826-f7e6d180-49bc-11ea-8ea9-7b67ef377b33.png)

Reply:
Good catch, thanks Ravi. Fixed now.